### PR TITLE
icons: replace react wrapped svgs with icons

### PR DESCRIPTION
**Pull request by curioussavage**
This includes: post, search, hamburger, vote
and seashell icons.

:eyeglasses: @dwick

### CODE REVIEW

File: assets/less/components/TopNav.less [hunk 1]
```diff
@@ -1,3 +1,5 @@
+@hamburger-width: 45px;
```
Comment (dwick) [inline assets/less/components/TopNav.less]: this is the same as `@top-nav-height` also, variables should be in the variables file if this is actually needed for some reason

**Comment by dwick** [comment]
remove the components too and cleanup the css!

**Comment by dwick** [comment]
:nail_care:

**Comment by curioussavage** [comment]
:haircut:  should be better now

File: assets/less/components/Dropdown.less [hunk 2]
```diff
@@ -9,6 +9,11 @@
     line-height: 20px;
   }
 
+  .icon-seashells {
```
Comment (dwick) [inline assets/less/components/Dropdown.less]: `.SeashellIcon` above is no longer used. 

Also, i'm not sure how @ajacksified feels about this, but I would generally just add a less presentational class for making small adjustments like you're doing here and remove the nesting.

Example:

```html
<span class="icon-seashells Dropdown-toggle"></span>
```

```css
.Dropdown {
  // ...
}
.Dropdown-toggle {
  display: inline-block;
  font-size: 2em;
}
```

**Comment by dwick** [comment]
I believe the css for `.SeashellIcon`, `.UpvoteIcon`, `.SearchIcon`, `.PostIcon`, `.HamburgerIcon` is still here.   Can you remove that too?

:nail_care:

**Comment by curioussavage** [comment]
:haircut:  that should be the last of the refs to the svg icons that were removed.

File: assets/less/components/TopNav.less [hunk 3]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (ajacksified) [inline assets/less/components/TopNav.less]: nuke newline

**Comment by ajacksified** [comment]
I do rather like the animated up/downvote icons, which differ from the rest in that you actually see them animate when clicking; what's the performance impact of leaving them in place?

**Comment by ajacksified** [comment]
:nail_care:

**Comment by ajacksified** [comment]
:fish: after talking in person about icons

**Pull request closed without merging**